src/pages/RFOX/components/Claim/ClaimRow.tsx

```diff
@@ -69,49 +95,57 @@ export const ClaimRow: FC<ClaimRowProps> = ({
           : 'RFOX.tooltips.unstakePendingCooldown',
         { cooldownPeriodHuman },
       )}
+      isDisabled={displayClaimButton}
     >
       <Flex
-        as={Button}
         justifyContent={'space-between'}
-        mt={2}
         align='center'
-        variant='unstyled'
-        p={8}
+        p={2}
         borderRadius='md'
+        height='auto'
         width='100%'
-        onClick={handleClaimClick}
-        isDisabled={status !== ClaimStatus.Available}
-        _hover={hoverProps}
+        {...parentProps}
       >
         <Flex>
-          <Box mr={4}>
+          <Flex alignItems='center' mr={4}>
             <AssetIconWithBadge assetId={foxAssetId}>
               <TransactionTypeIcon type={TransferType.Receive} />
             </AssetIconWithBadge>
-          </Box>
+          </Flex>
           <Box mr={4}>
             <RawText fontSize='sm' color='gray.400' align={'start'}>
-              {translate('RFOX.claim', { assetSymbol: stakingAssetSymbol })}
+              {actionDescription
+                ? actionDescription
```

Comment:
Also do we want to pass this as a translation key instead? i.e its only usage is:

<img width="1573" alt="image" src="https://github.com/shapeshift/web/assets/17035424/761ed17a-6db0-4060-bd47-5ed200bf5064">

This way we. could extract the translation key into its own const i.e "if actionDescription, return description, if `stakingAssetSymbol` return claim translation" then use that inside `translate()`

Though writing this gets me wondering about the API of this : according to this ternary, it looks as if `actionDescription` is what dictates whether this is a claim button or description translation variant, however, `displayClaimButton` is when looking at consumptions.

Do we want to use a discriminated union for the props, so we can do the ternary (or whatever control flow we use if abstracting the translation away) on `displayClaimButton`

Reply:
Added the union type and based this ternary on `displayClaimButton` in [683d93f](https://github.com/shapeshift/web/pull/7098/commits/683d93fbc32c413ab11ded7c3829d3d96e3720c9)